Gemfile

```diff
@@ -1,4 +1,2 @@
 source 'https://rubygems.org'
 gemspec
-
-gem 'redis'
```

Comment:
> I'd rather avoid using ENV vars for installation. That can make things more complicated for users, and libs upstream that use Classifier.

I agree! However, I have seen this trick being used in [SciRuby/rb-gsl/Gemfile](https://github.com/SciRuby/rb-gsl/blob/master/Gemfile).